tsconfig.json

```diff
@@ -7,10 +7,9 @@
     "skipLibCheck": true,
 
     /* Bundler mode */
-    "moduleResolution": "bundler",
-    "allowImportingTsExtensions": true,
+    "moduleResolution": "Node",
     "resolveJsonModule": true,
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
This will break the build `npm run build` I believe. Was this really necessary to run `npm run generate-abilities`? If I remember correctly generating abilities file worked with the previous config.

Reply:
Let's not change it then